go/extra/stats/cmd/stats.go

```diff
@@ -293,11 +293,9 @@ func getStats(ctx context.Context, consensus consensusAPI.ClientBackend, registr
 		}
 
 		// Go over all signatures for a block.
-		// XXX: In tendermint master (not yet released) the signatures are
-		// obtained in LastCommit.Signatures.
-		for _, sig := range tmBlockMeta.LastCommit.Precommits {
-			if sig == nil {
-				logger.Debug("skipping nil-votes")
+		for _, sig := range tmBlockMeta.LastCommit.Signatures {
+			if sig.Absent() {
```

Comment:
@ptrus can you confirm that this is ok?

Reply:
yeah seems like `nil` votes match the new `Absent()` votes, so seems fine :+1: